lib/plain/plain-client.ts

```diff
@@ -413,37 +412,36 @@ export const createPlainClient = (
       get: wrap(wrapParams, 'UserUIConfig', 'get'),
       update: wrap(wrapParams, 'UserUIConfig', 'update'),
     },
-    ...addAlphaFeatures(makeRequest, defaults, alphaFeatures),
-  }
-}
-
-const addAlphaFeatures = (
-  makeRequest: MakeRequest,
-  defaults: DefaultParams | undefined,
-  alphaFeatures?: string[]
-): AlphaExtensions | Record<string, never> => {
-  const wrapParams = { makeRequest, defaults }
-  const alphaInterface: AlphaExtensions = {} as AlphaExtensions
-
-  if (alphaFeatures?.includes(ALPHA_FEATURE_WORKFLOWS)) {
-    alphaInterface.workflowDefinition = {
+    workflowDefinition: {
       get: wrap(wrapParams, 'WorkflowDefinition', 'get'),
       getMany: wrap(wrapParams, 'WorkflowDefinition', 'getMany'),
       create: wrap(wrapParams, 'WorkflowDefinition', 'create'),
       update: wrap(wrapParams, 'WorkflowDefinition', 'update'),
       delete: wrap(wrapParams, 'WorkflowDefinition', 'delete'),
-    }
-    alphaInterface.workflow = {
+    },
+    workflow: {
       getMany: wrap(wrapParams, 'Workflow', 'getMany'),
       create: wrap(wrapParams, 'Workflow', 'create'),
       update: wrap(wrapParams, 'Workflow', 'update'),
       delete: wrap(wrapParams, 'Workflow', 'delete'),
       complete: wrap(wrapParams, 'Workflow', 'complete'),
-    }
-    alphaInterface.workflowsChangelog = {
+    },
+    workflowsChangelog: {
       getMany: wrap(wrapParams, 'WorkflowsChangelog', 'getMany'),
-    }
+    },
+    ...addAlphaFeatures(makeRequest, defaults, alphaFeatures),
   }
+}
+
+const addAlphaFeatures = (
```

Comment:
It's not in use and I'm not sure if it will be, but keeping it for the future as a nice thing, tell me if you wait it to be removed as well.

Reply:
I'm not sure if we want to tackle alpha features in the future again in this repo and in the way I implemented it initially. IMHO it would be good to drop it. @andipaetzold wdyt?

Reply:
I've updated the PR to remove all alpha features support.

Reply:
Imo we should release alpha features as canary versions of CMA.js. Anything available in `latest` will be used by our users, even if it says "alpha".